### PR TITLE
list component added

### PR DESCRIPTION
**Pull request by MonikaJhingan**
Please review urgently.

### CODE REVIEW

File: List/list.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.list-item {
+  width: 50rem;
+  font-size: 3rem;
+  margin-left: 40rem;
+  padding: 1rem;
+  border-bottom: 2px solid var(--success-color);
+}
+
+.list-item:hover {
+  background-color: rgba(40, 209, 187, 0.2);
```
Comment (pratik-kumar-sahu) [inline List/list.css:21]: - use color as variables

**Review by pratik-kumar-sahu — APPROVED** [review]
Else looks good to be merged

File: List/list.html [hunk 2]
```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (ranitasahaa1230) [inline List/list.html:6]: You can add desc in the meta tag, like author name, page desc, page details etc

File: List/list.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="../../Utility CSS/utility.css" />
+    <link rel="stylesheet" href="./list.css" />
+
+    <link rel="icon" href="../../img/logo.png" type="image/x-icon" />
+    <title>Card</title>
```
Comment (ranitasahaa1230) [inline List/list.html]: Please change the title to List, before merging it to dev branch

**Review by ranitasahaa1230 — APPROVED** [review]
Rest looks good to me. Good to merge

File: List/list.css (hunk 1 above)
Comment (MonikaJhingan) [reply List/list.css:21]: will ensure that

File: List/list.html (hunk 2 above)
Comment (MonikaJhingan) [reply List/list.html:6]: yes

File: List/list.html (hunk 3 above)
Comment (MonikaJhingan) [reply List/list.html]: ok

**Pull request merged**